### PR TITLE
Fix DBC Load and updated several smaller things

### PR DESCRIPTION
**Pull request by funjoker**
- Copyright Update
- Hotfix Parsing
- DB2Hash update

Thanks goes to @wowdev for DBC library

### CODE REVIEW

**Comment by Shauren** [comment]
Small think? Nice

**Comment by funjoker** [comment]
How to make this properly working for travis and co?
DBFileReaderLib is a NET Standard lib.

File: WowPacketParser/WowPacketParser.csproj [hunk 1]
```diff
@@ -91,6 +91,7 @@
     <LangVersion>6</LangVersion>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include=”netstandard” />
```
Comment (Shauren) [inline WowPacketParser/WowPacketParser.csproj]: You will not make any build system happy with these weird language specific quote marks

Comment (funjoker) [reply WowPacketParser/WowPacketParser.csproj]: already noticed it xD

**Comment by funjoker** [comment]
i hate mixing stuff... need help here

**Pull request closed without merging**